app/server/endpoint/src/test/java/io/syndesis/server/endpoint/v1/handler/meta/AggregateMetadataHandlerTest.java

```diff
@@ -183,22 +161,7 @@ public void shouldExtractJsonSchemaInputElementVariant() throws IOException {
                                 .build())
                         .addVariant(dummyShape(DataShapeKinds.JAVA))
                         .build())
-                .outputShape(new DataShape.Builder()
-                        .kind(DataShapeKinds.JSON_SCHEMA)
-                        .specification(getSpecification("person-schema.json"))
-                        .description("person-schema")
-                        .type(Person.class.getName())
-                        .putMetadata(StepMetadataHandler.VARIANT_METADATA_KEY, StepMetadataHandler.VARIANT_ELEMENT)
-                        .addVariant(new DataShape.Builder()
-                                .kind(DataShapeKinds.JSON_SCHEMA)
-                                .specification(getSpecification("person-list-schema.json"))
-                                .collectionType("List")
-                                .type(Person.class.getName())
-                                .collectionClassName(List.class.getName())
-                                .putMetadata(StepMetadataHandler.VARIANT_METADATA_KEY, StepMetadataHandler.VARIANT_COLLECTION)
-                                .build())
-                        .addVariant(dummyShape(DataShapeKinds.JAVA))
-                        .build())
+                .outputShape(StepActionHandler.NO_SHAPE)
```

Comment:
Perhaps add a new test case for `NO_SHAPE` and adapt the test for `JSON_SCHEMA`?

Reply:
The test was an actual duplicate with the next test variant where the exact same output shape was used and verified. I changed that so we have input and output shape tested in separate tests